docs/products/postgresql/howto/migrate-aiven-db-migrate.rst

```diff
@@ -37,10 +43,10 @@ If the output is not ``logical``, run the following command in ``psql`` and then
 .. Note::
     If you are migrating from an AWS RDS PostgreSQL cluster, set the ``rds.logical_replication`` parameter to ``1`` (true) in the parameter group.
 
-Check the Migration Configuration Options
-'''''''''''''''''''''''''''''''''''''''''
+Checking the migration configuration options
```

Comment:
Again, imperative rather than Gerund - but definitely :+1: for standardising on sentence case for titles.